CHANGELOG.md

```diff
@@ -15,6 +15,12 @@ Unreleased
 * Add theme support
 * Changed default branch from master to main.
 
+### Removed
```

Comment:
oops these need to be moved under unreleased section

Reply:
I think that accidentally got moved during a main merge, fixed [here](https://github.com/cerner/terra-dev-site/pull/281/commits/350c0ed9d94afee7893917bab06e6d219bc195de).